docs/framework/guides/contributing/git-commit-message-convention.md

```diff
@@ -47,23 +47,33 @@ Each commit can contain additional notes which will be inserted into the changel
 | Type | Is backward compatible? |
 | --- | --- |
 | `NOTE` | Yes |
-| `BREAKING CHANGES` or `BREAKING CHANGE` | No |
+| `MAJOR BREAKING CHANGE`, `MINOR BREAKING CHANGE` or `BREAKING CHANGE` | No |
 
-If any visible change contains the `BREAKING CHANGE` note, the next release will be marked as `major` automatically.
+The `BREAKING CHANGE` is an alias to a `MAJOR BREAKING CHANGE`.
+
+If any change contains the `MAJOR BREAKING CHANGE` note, the next release will be marked as `major` automatically.
+
+For reference on how to identify minor or major breaking change see the {@link framework/guides/support/versioning-policy versioning policy guide}.
+
+### Package name
+
+Most commits are related to one or more packages. Each affected package should be listed in parenthesis following the commit type. A package that was the most impacted by the change should be listed first.
+
+It is, however, possible to skip this part if five or more packages are affected.
 
 ### Example commits
 
 A new feature without any breaking changes.
 
 ```
-Feature: Added support for RTL languages. Closes #1.
+Feature (ui, theme-lark): Added support for RTL languages. Closes #1.
```

Comment:
That's incorrect: https://github.com/ckeditor/ckeditor5/issues/6058#issuecomment-611482923.